### ISSUE TITLE
Suggestion: Add % chance of encountering an opponent hero stats

### ISSUE THREAD

**Issue by anonymous555** [HearthSim/Hearthstone-Deck-Tracker#464]
To help with identifying the meta. 

In the overall stats, show what percent of the time we play against a given hero.
Say, 38% shamen, 21 % warlock etc.  And like the other stats, have it sorted by time frame.

**Comment by Elitesparkle** [comment]
In the Overall tab, when the "Show as percent" check box is checked it should show the percentage of the games you have played against a given hero, when not it should show the number of games you have played against that given hero.

Also, I'd add that check box in the Deck tab as well.

**Comment by Bahzur** [comment]
Wait, arent these the win-lose percentages you described?

**Comment by Elitesparkle** [comment]
Sure. I was just clarifing that, since there is that check box, it could show both percentage (when checked) and number (when unchecked) of games played.

**Comment by Bahzur** [comment]
But... the percentages shown dont have anything to do with the number of times you encountered that hero! They only say how often youve won against that hero in any given timeframe.
Yes, when you uncheck percentages, you see how many games you played against that hero if you add your wins-losses together, but the point of this feature request is another!

**Comment by Elitesparkle** [comment]
No, let me clarify. I'm suggesting the add a new row to each table to show those values.

**Comment by Bahzur** [comment]
Im stupid. I overread your "should" and "could" both times. Sry :(

**Comment by Elitesparkle** [comment]
No problem.

**Comment by Cu8eR** [comment]
+1, it would be awesome. One problem - it needs to be reseted every new season

**Comment by GiggliG** [comment]
It doesn't necessarily need to be reset each season. It should probably use the same time period as whatever time filter you have chosen.
